modules/express/src/typedRoutes/api/v2/ofcSignPayload.ts

```diff
@@ -0,0 +1,45 @@
+import * as t from 'io-ts';
+import { Json, NonEmptyString, JsonFromString } from 'io-ts-types';
+import { httpRoute, httpRequest, optional } from '@api-ts/io-ts-http';
+import { BitgoExpressError } from '../../schemas/error';
+
+/**
+ * Sign an arbitrary payload using an OFC trading account key.
+ */
+export const OfcSignPayloadBody = {
+  /** The ID of the OFC wallet to sign the payload with. */
+  walletId: NonEmptyString,
+  /** The payload to sign. Will always decode into a JSON object. */
```

Comment:
```suggestion
  /** The payload to sign. The input can either be a stringified JSON, or a JSON object. */
```

These will eventually go into out api docs, lets write them form the perspective of an integrator for this api.